packages/x-components/src/x-modules/recommendations/components/recommendations.vue

```diff
@@ -73,6 +73,14 @@
     @Prop({ default: 'ul' })
     protected animation!: Vue;
 
+    /**
+     * Number of recommendations to be rendered.
+     *
+     * @public
+     */
+    @Prop({ default: 5 })
```

Comment:
I would not set a default max items to render for this component. Standard use case is to render all available recommendations , and we can't set `undefined` in the template because it will fallback to this default value.